src/data/graphql/Database/battle/GetBattletime.js

```diff
@@ -30,42 +30,60 @@ export const queries = [
 export const resolvers = {
   RootQuery: {
     async getBattletime(parent, { BattleIndex }) {
-      const battletime = await Battletime.findAll({
+      const battleStatus = await Battle.findAll({
+        attributes: ['Finished'],
         where: { BattleIndex },
-        include: [
-          {
-            model: Kuski,
-            attributes: ['Kuski', 'Country'],
-            as: 'KuskiData',
-            include: [
-              {
-                model: Team,
-                as: 'TeamData',
-              },
-            ],
-          },
-        ],
       });
+      let battletime;
+      if (battleStatus[0].dataValues.Finished === 1) {
```

Comment:
This is a bit pointless isn't it since battletime table contains the times only after the battle is finished? It makes sense when reading from allfinished though then it should check if the battle is hidden times and not if it is finished?

Reply:
It's hard to know with the test database, but this query releases results of the ongoing and queued battles (ids 124967 & 124968). Maybe the server only updates results tables after battles are finished and they're just artificially filled out here (because the ongoing and queued battles were set arbitrarily), but I don't know how I could test that without deploying gak to live db. If it's as you say then it would save me some work with the GetBattle.js queries, so if you know how to make sure about that then please let me know.

Agree that eventually it should hide times only for ongoing battles with hidden times, but I don't know how to fetch hidden times status yet, this was meant to be a temporary filter that prevents unwanted leakage of data.

Reply:
See here as well:
https://github.com/elmadev/elmaonline-site/issues/121

Please comment if I don't understand something or if something I say is wrong.

Reply:
Oh right. Well the database [sheet](https://docs.google.com/spreadsheets/d/15fNKf2ihV4HvmVZwxg2D18ITvcbCE1nva5NTFlYJOgg/edit?usp=sharing) says it's updated once the battle ends and also [https://elma.online/battles](https://elma.online/battles) has never shown times for ongoing battles so I'd say it's safe. Allfinished is supposedly updated every 5 minutes, maybe this information should be clearly visible somewhere.

Reply:
Thanks for that! I'll look into it later. Unfortunately it looks like I won't be able to do anything more before I leave for a month, but if what you say is true then my current PR should be safe and good to go.

Reply:
Alright cool. I was going to work on showing the live results on ongoing battle on the site anyway so I can continue working on top of this once it's merged.